packages/renderer/src/lib/preferences/PreferencesRegistriesEditing.svelte

```diff
@@ -0,0 +1,459 @@
+<style>
+.dropdown:hover .dropdown-menu {
+  display: block;
+}
+</style>
+
+<script lang="ts">
+import type { Registry } from '@tmpwip/extension-api';
+import { onMount } from 'svelte';
+import { registriesInfos } from '../../stores/registries';
+
+let registryStore: Registry[] = [];
+onMount(() => {
+  registriesInfos.subscribe(value => {
+    registryStore = value;
+  });
+});
```

Comment:
here as we just need the current value we can use the `$` alias to avoid also to unsubscribe on `onDestroy`

so instead of declaring `registryStore` and using it, you can just use instead `$registriesInfos `

Reply:
updated